app/backend/graphql/team/team.ts

```diff
@@ -29,5 +30,21 @@ export const Team = objectType({
       description: 'List of all customers of the team',
       resolve: (team, _arguments, context) => context.prisma.customer.findMany({ where: { teamId: team.id } }),
     })
+    t.list.field('projects', {
+      type: Project,
+      description: 'List of all projects of the team',
+      resolve: (team, _arguments, context) =>
+        context.prisma.project.findMany({
+          where: {
+            AND: {
+              teamId: team.id,
+              OR: {
+                projectMemberships: { some: { userId: context.session?.user.id } },
+                team: { teamMemberships: { some: { role: 'ADMIN', userId: context.session?.user.id } } },
```

Comment:
If the session is undefined, then these where clause will not be checked.
The case is rather unlikely, but if we can avoid it, maybe we should do it.

Reply:
done